homeassistant/components/tplink/number.py

```diff
@@ -51,6 +51,10 @@ class TPLinkNumberEntityDescription(
         key="temperature_offset",
         mode=NumberMode.BOX,
     ),
+    TPLinkNumberEntityDescription(
+        key="power_protection_threshold",
+        mode=NumberMode.BOX,
```

Comment:
```suggestion
        mode=NumberMode.SLIDER,
```
I think this will work better as a slider.